lib/app.js

```diff
@@ -102,15 +101,35 @@ class App {
       if      (shell.test('-e', oldTrufflePath)) truffleConfig = reqCwd.silent(oldTrufflePath);
       else if (shell.test('-e', newTrufflePath)) truffleConfig = reqCwd.silent(newTrufflePath);
 
+      const coverageNetwork = {
+        host: 'localhost',
+        network_id: '*',
+        port: this.port,
+        gas: gasLimitHex,
+        gasPrice: gasPriceHex
+      };
+
       // Coverage network opts specified: use port if declared
       if (truffleConfig && truffleConfig.networks && truffleConfig.networks.coverage) {
         this.port = truffleConfig.networks.coverage.port || this.port;
         this.network = '--network coverage';
 
       // No coverage network defaults to the dev network on port 8555, high gas / low price.
+      } else if (truffleConfig) {
```

Comment:
Here, can we make this just plain `else`? Truffle always needs to find a config anyway so it's probably there. To be safe we could initialize truffleConfig to `{}` above here where it's declared.

Reply:
And if truffleConfig is initialized by default the condition check above this else can be simplified to:
```javascript
if (truffleConfig.networks && truffleConfig.networks.coverage)
```